app/scripts/templates/sign_in_totp_code.mustache

```diff
@@ -28,8 +28,9 @@
 
     </form>
 
-    <div class="links">
-        <a id="use-recovery-code-link" class="center" href="/signin_recovery_code">{{#t}}Use recovery code{{/t}}</a>
+    <div class="links centered">
+      <a id="use-recovery-code-link" class="center" href="/signin_recovery_code">{{#t}}Use recovery code{{/t}}</a>
```

Comment:
The `class="center"` isn't needed because `centered` specified in the div above.